corehq/apps/enterprise/api/resources.py

```diff
@@ -408,6 +408,19 @@ def get_primary_keys(self):
         return ('form_id', 'submitted',)
 
 
+class TwoFactorAuthResource(ODataEnterpriseReportResource):
+    domain_without_2fa = fields.CharField()
```

Comment:
Looking directly below, at the `CommCareVersionComplianceResource`, it does seem like we use `project_space` for any user-facing things,so `domain_without_2fa` should be changed to `project_space_without_2fa`?

Reply:
`CommCareVersionComplianceResource` might be an exception, all other resources are using `domain`... Maybe it is `CommCareVersionComplianceResource` need to use `domain` ?

Reply:
Yep, good catch.  We should probably update `CommCareVersionComplianceResource`, but it looks like this one is good to go